lib/rules/declaration-block-no-shorthand-property-overrides/index.js

```diff
@@ -18,8 +18,13 @@ const rule = function (actual) {
       return
     }
 
-    root.walkRules(check)
-    root.walkAtRules(check)
+    root.walk((node) => {
+      if (node.type !== "rule" && node.type !== "atrule") {
+        return
+      }
+
+      check(node)
+    })
```

Comment:
Is this change the only thing that caused the slight speed-up? Does such a change reliably cause speed-ups in other rules? What we had before matches a pattern that is widely used throughout the rules, so I'd prefer to stick with it unless we get some significant benefit from changing it.